docs/documentation.md

```
@@ -427,16 +427,17 @@ pull-request. This requires you already have a GitHub account.
 
   ```none
   $ git fetch upstream
-  $ git checkout current
-  $ git merge upstream/current
-```
+  $ git checkout rolling
```

markdownlint-cli2 (0.22.1)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)